config/locales/en.yml

```diff
@@ -1550,7 +1550,7 @@ en:
       cinstance:
         intentions: "Please describe briefly your intentions of use of this service"
         user_key: "Allowed characters: [A-Z a-z 0-9 - _ .], or Base64 format without forward slash (/), no spaces and up to 256 characters."
-        key: "Allowed characters: [A-Z a-z 0-9 ! \" # $ % & ' ( ) * + , - . / : ; < = > ? @ [ \ ] ^ _ ` { | } ~], no spaces and between 5 and 255 characters."
+        key: "Allowed characters: [A-Z a-z 0-9 ! \" # $ % & ' ( ) * + , - . / : ; < = > ? @ [ \\ ] ^ _ ` { | } ~], no spaces and between 5 and 255 characters."
```

Comment:
why do we put characters inside `[]`? And then `[` and [`] are also inside these brackets...

Reply:
This is only a hint for the user, as long it's understandable, I think it's fine.

Reply:
I removed the square brackets, and I think it looks fine:

![Screenshot from 2024-11-27 13-12-19](https://github.com/user-attachments/assets/d4240762-d653-4fe7-96db-d61364430d03)

![Screenshot from 2024-11-27 13-12-55](https://github.com/user-attachments/assets/dfceffe4-095a-42d5-973d-fd752a60cee5)